### PR TITLE
made changes for FHIR-39512

### PR DESCRIPTION
**Pull request by jdlnolen**
## HL7 FHIR Pull Request

_Note: No pull requests will be accepted against `./source` unless logged in the_ [HL7 Jira issue tracker](https://jira.hl7.org/projects/FHIR/issues/).

If you made changes to any files within `./source` please indicate the Jira tracker number this pull request is associated with: `   `

## Description

made changes for FHIR-39512

### CODE REVIEW

File: source/servicerequest/servicerequest-request-mapping-exceptions.xml
Comment (davidbarwin-stemsoft) [inline source/servicerequest/servicerequest-request-mapping-exceptions.xml:1]: Is this change intended for this JIRA?

Comment (jdlnolen) [reply source/servicerequest/servicerequest-request-mapping-exceptions.xml:1]: I think the publisher made that change. Not me directly.
